tests/Telemetry/ChatTelemetryTests.cs

```diff
@@ -175,11 +155,10 @@ public void ChatTracingException()
     [Test]
     public async Task ChatTracingAndMetricsMultiple()
     {
-        using var _ = TestAppContextSwitchHelper.EnableOpenTelemetry();
         var source = new OpenTelemetrySource(RequestModel, new Uri(Endpoint));
 
-        using var activityListener = new TestActivityListener("OpenAI.ChatClient");
-        using var meterListener = new TestMeterListener("OpenAI.ChatClient");
+        using var activityListener = new TestActivityListener("Experimental.OpenAI.ChatClient");
```

Comment:
I wish everyone uses OTel's InMemoryExporter rather than writing own listeners, but it is upto the owners of this project!

Reply:
can you explain why?

Reply:
Listeners are not extensively documented given its generally only used by small set of people like people authoring OpenTelemetry SDK etc.
OTOH, InMemoryExporter is documented, and is intended to be used by end-users to validate their own instrumentation.  Changing from InMemoryExporter to anything else like Console/OTLP is often the one-line change. End-users, who are familiar with OTLP Exporter find it easy to use InMemoryExporter with OTel for testing, as opposed to learning ActivityListener/MeterListener etc.

Just a suggestion only.